index.html

```diff
@@ -229,64 +229,127 @@ <h2 class="title">
+            </h3>
+            <ul>
+              <li>Javascript</li>
+              <li>Ruby</li>
+              <li>Html</li>
+              <li>CSS</li>
+            </ul>
           </div>
-        </div>
-        <div class="content-exp">
-          <h3>
-            Frameworks
-          </h3>
-          <ul>
-            <li>Bootstrp</li>
-            <li>Ruby on rails</li>
-            <li>RSepec</li>
-            <li>Capybara</li>
-            <li>Selenium</li>
-          </ul>
-        </div>
-      </li>
-      <li class="card-exp">
-        <div class="profile-exp">
-          <div class="image">
-            <img src="images/tools.png" alt="" height="44" width="44">
+        </li>
+        <li class="card-exp">
+          <div class="profile-exp">
+            <div class="image">
+              <img src="images/framework.png" alt="" height="44" width="44">
+            </div>
           </div>
-        </div>
-        <div class="content-exp">
-          <h3>
-            Skills
-          </h3>
-          <ul>
-            <li>Codekit</li>
-            <li>GitHub</li>
-            <li>Codepen</li>
-            <li>GitLab</li>
-            <li>Termin</li>
-          </ul>
-        </div>
-      </li>
-    </ul>
+          <div class="content-exp">
+            <h3>
+              Frameworks
+            </h3>
+            <ul>
+              <li>Bootstrp</li>
+              <li>Ruby on rails</li>
+              <li>RSepec</li>
+              <li>Capybara</li>
+              <li>Selenium</li>
+            </ul>
+          </div>
+        </li>
+        <li class="card-exp">
+          <div class="profile-exp">
+            <div class="image">
+              <img src="images/tools.png" alt="" height="44" width="44">
+            </div>
+          </div>
+          <div class="content-exp">
+            <h3>
+              Skills
+            </h3>
+            <ul>
+              <li>Codekit</li>
+              <li>GitHub</li>
+              <li>Codepen</li>
+              <li>GitLab</li>
+              <li>Termin</li>
+            </ul>
+          </div>
+        </li>
+      </ul> 
+    </section>
   </main>
+  <footer>
+    <div class="footer-content">
+      <p class="title pad">
+        I'm always interested in hearing about new projects, so if you'd like
+        to chat please get in touch.
+      </p>
+    </div>
+    <div class="contact-form">
+      <form action="https://formspree.io/f/meqnbakq" method="POST" >
+        <ul>
+          <li>
+            <input type="text" name="name" placeholder="Full name" maxlength="30" required>
+          </li>
+          <li>
+            <input type="email" name="email" placeholder="Email address" required>
+          </li>
+          <li class="text-border">
+            <textarea required maxlength="500" name="message" cols="30" rows="10" placeholder="Enter text here"></textarea>
+          </li>
+          <li>
+            <button type="submit" class="button">
```

Comment:
- [ ] I kindly suggest you add the correct button interactions for active (when it's pressed) because it is a project requirement.

![image](https://user-images.githubusercontent.com/52934128/177439211-58ea3e6c-bf4e-493e-863d-9c6ce07d7d1e.png)



the button on the pressed state has a darker `background: #008552;`

### EXAMPLE:
**for pressed state**
```
button:active {
   background-color: #008552;
}
```
**for disabled state**
```